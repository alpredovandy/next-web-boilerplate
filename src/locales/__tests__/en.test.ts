import commonEn from "../en";

describe("Test locales: en", () => {
  it("Snapshot", () => {
    expect(commonEn).toMatchSnapshot();
  });
});
