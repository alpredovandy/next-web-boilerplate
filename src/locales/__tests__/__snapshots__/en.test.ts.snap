// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test locales: en Snapshot 1`] = `
{
  "common": {
    "validation": {
      "required": "{name} must be filled.",
    },
  },
  "components": {
    "calendar": {
      "acceptText": "Apply",
      "cancelText": "Cancel",
    },
    "timePicker": {
      "cancelText": "Cancel",
      "labels": [
        "Hour",
        "Minutes",
      ],
      "toolbarTitle": "Select time",
    },
  },
  "fiveHundred": {
    "action": "WhatsApp",
    "description": "Rey's system is experiencing problems. Please try again in a few moments or contact us via",
    "title": "Sorry, System Error",
  },
  "fourOFour": {
    "action": "Home",
    "description": "This page does not exist on Rey. Come on, double-check your destination link or return to",
    "title": "Well, Page Not Found",
  },
}
`;
